tests/integration_category_colors/Frontend/__snapshots__/CSS_Generator_Test__should_handle_hidden_categories__1.php

```diff
@@ -1,12 +1,11 @@
-<?php return '.tribe_events_cat-visible-category {
-    --tec-color-category-primary: #ff0000;
-    --tec-color-category-secondary: #00ff00;
-    --tec-color-category-text: #0000ff;
-}
-
-.tribe_events_cat-hidden-category {
+<?php return '.tribe_events_cat-hidden-category {
     --tec-color-category-primary: #0000ff;
     --tec-color-category-secondary: #00ff00;
     --tec-color-category-text: #ff0000;
 }
+.tribe_events_cat-visible-category {
```

Comment:
This change is weird. @redscar is this expected?

Reply:
@Camwyn It look's like the order of styles changed, the data is the same. I'm not sure why that is though. If the category doesn't have a priority it may order it incorrect, which could be the case here. I'll have to look at the test and possibly add a priority to the category.